server/src/main/java/com/hedera/block/server/Constants.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Hedera Block Node
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.server;
+
+public final class Constants {
+    private Constants() {}
+
+    // Config Constants
+    public static String BLOCKNODE_STORAGE_ROOT_PATH_KEY = "blocknode.storage.root.path";
```

Comment:
If these constants are only used from a single place, we should relocate and make private in the class it is used in. If it is used in multiple places, then we should think about our config strategy, so that everyone just has a POJO to get values from instead of having to know the path key.

Reply:
I was following Helidon's suggested strategy of making their merged config object global:
```
final Config config = Config.create();
Config.global(config);
```
Then you use their config obj + key to get the values you need.  It looks like theirs supports a deterministic set of rules to determine property precedent (env props take precedent over yaml props, etc).  Are you suggesting a different approach?

Reply:
We can revisit this later